docs/source/tools/visual.rst

```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
 1. Install `Graphviz <http://www.graphviz.org/download/>`_.
 2. Generate a DAR from your project by running ``daml build``.
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
-4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
\ No newline at end of file
+4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``.
+
+
+Visual for Quickstart
```

Comment:
I'd change this to something more descriptive, eg
```suggestion
Example: visualizing the Quickstart project
```

You'll need to update the line beneath so the title underline is long enough.